homeassistant/components/google_assistant_sdk/strings.json

```diff
@@ -24,7 +24,8 @@
       "unknown": "[%key:common::config_flow::error::unknown%]",
       "oauth_timeout": "[%key:common::config_flow::abort::oauth2_timeout%]",
       "oauth_unauthorized": "[%key:common::config_flow::abort::oauth2_unauthorized%]",
-      "oauth_failed": "[%key:common::config_flow::abort::oauth2_failed%]"
+      "oauth_failed": "[%key:common::config_flow::abort::oauth2_failed%]",
+      "single_instance_allowed": "[%key:common::config_flow::abort::single_instance_allowed%]"
```

Comment:
I think this might conflict with work from @mib1185

Reply:
Maybe investigate first why it isn't set in the manifest?

Reply:
Yes, you are right. Closed the PR now.